### PR TITLE
Incentivize faucet transaction per number of outputs

### PR DESCRIPTION
**Pull request by deekerno**
## Summary
We should incentivize the inclusion of faucet transactions in a block by increasing the fee awarded per output.

## Testing Plan
Manual testing and calculation.

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by dguenther — APPROVED** [review]
This looks good, I'm going to hold off on merging it until we finish the in-progress release, since that staging branch hasn't been merged to master yet.

**Comment by dguenther** [comment]
This needs a rebase/cleanup before merge

**Comment by deekerno** [comment]
> This needs a rebase/cleanup before merge

Done.

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**
